cartridge-cli.lua

```diff
@@ -845,6 +905,13 @@ local POSTBUILD_SCRIPT_NAME = 'cartridge.post-build'
 local DEP_PREBUILD_SCRIPT_NAME = '.cartridge.pre'
 local DEP_IGNORE_FILE_NAME = '.cartridge.ignore'
 
+-- build directory
+
+local HOME_DIR = os.getenv('HOME') or '/home'
```

Comment:
```bash
> env
...
HOME=/Users/o.babin
...
```

I think it should be `~/` instead of `/home`

Reply:
`fio` thinks that `~` is a `~` directory in the local path. E.g. if you call `fio.abspath('~')` from `/path/to/me`, you get `/path/to/me/~`.

Reply:
looks like a fio bug

Reply:
but ok. Let's use `env HOME`